src/app/_components/FocusAreaCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { Heading } from '@/components/Heading'
```

Comment:
I'm tempted to say that we could use `Card` but I'm not sure without trying?

Reply:
The layout changes from vertical to horizontal and back to vertical, depending on the breakpoint. I don't think it's possible to do with the current implementation of `Card`?

There will be a proper refactor of the card component eventually to break it down into smaller pieces and make these layouts possible :)

![CleanShot 2024-05-31 at 14 28 57](https://github.com/FilecoinFoundationWeb/filecoin-foundation/assets/20277959/48d546f5-1f9e-4544-93b4-19e301cfb9ca)